recipes-ni/ni-hw-support/ni-hw-support-common_1.0.bb

```diff
@@ -0,0 +1,54 @@
+SUMMARY = "Support scripts and utilities common to all NI hardware products"
```

Comment:
Just occurred to me that ni-hw-integration could be another alternative if you weren't crazy about ni-hw-scripts. Very similar to ni-hw-support, but maybe without quite the same implication of driver support?

Reply:
I would prefer `-scripts` to `-integration`.